fsd_utils/simple_utils/date_utils.py

```diff
@@ -7,21 +7,17 @@ def current_datetime_after_given_iso_string(value: str) -> bool:
     # Grab local timezone
     uk_tz = pytz.timezone("Europe/London")
     # Grab current localised datetime
-    now_datetime = datetime.now(uk_tz)
-    # Convert now_datetime to offset-naive
-    now_datetime_naive = now_datetime.replace(tzinfo=None)
+    now_datetime = datetime.now(uk_tz).replace(tzinfo=None)
```

Comment:
im not 100 but this looks like its undoing itself

Reply:
so i understand your trying to make both times naive now (to compare), but it looks like in doing so the BST hour is being removed again

Reply:
yes you're 100% right, it does indeed look like it undoes itself, but what's actually going on is:
1) we're getting the current datetime localised so if we're in BST it will be an hour ahead.
2) by doing the above we also implicitly add the offset information to the datetime even though for the comparison we don't need this as it's already grabbed the localised time.
3) to be able to compare the 2 times (value and now()) we'll need them both to be of the same datetime format, but the now() has the offset information which needs to be removed.
4) so we remove the offset info using .replace(tzinfo=None) therefore allowing us to compare the 2 datetimes.

Reply:
perhaps something like:
```

def current_datetime_after_given_utc_string(value: str) -> bool:
    uk_tz = timezone("Europe/London")
    gmt_tz = timezone("GMT")

    parsed = datetime.strptime(value, "%Y-%m-%d %H:%M:%S").replace(tzinfo=gmt_tz)
    now_datetime = datetime.now(uk_tz)

    return now_datetime > parsed
```

Reply:
(the bst hour does not get removed when we move the offset info)

Reply:
Something like:
```

def current_datetime_after_given_utc_string(value: str) -> bool:
    uk_tz = timezone("Europe/London")
    parsed = datetime.strptime(value, "%Y-%m-%d %H:%M:%S").replace(tzinfo=timezone.utc)
    now_datetime = datetime.now(uk_tz)

    return now_datetime > parsed

```

The timezone.utc constant represents Coordinated Universal Time (UTC) with a fixed offset of 0.

Reply:
the problem with the above, is that the datetime is no longer offset naïve (as it's still a value even if its 0) therefore we wont be able to compare the 2 times.